core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -616,7 +683,7 @@ class KafkaZkClient private[zk] (zooKeeperClient: ZooKeeperClient, isSecure: Boo
     getDataResponses.flatMap { getDataResponse =>
       val topic = getDataResponse.ctx.get.asInstanceOf[String]
        if (getDataResponse.resultCode == Code.OK) {
-        val partitionMap = TopicZNode.decode(topic, getDataResponse.data).map { case (k, v) => (k.partition, v) }
+         val partitionMap = TopicZNode.decode(topic, getDataResponse.data).assignment.map { case (k, v) => (k.partition, v) }
```

Comment:
nit: indentation - looks like the indentation of the `if` statement above is off.

Reply:
I've found a few style errors while working on this and I wasn't sure the protocol. I guess it makes sense to fix the style in the block of code I'm working on. (But it's a little confusing when its not the areas of code I'm working on)